src/sentry/static/sentry/app/components/events/interfaces/keyValueList/keyValueListV2.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import sortBy from 'lodash/sortBy';
+import styled from '@emotion/styled';
+
+import ContextData from 'app/components/contextData';
+import AnnotatedText from 'app/components/events/meta/annotatedText';
+import theme from 'app/utils/theme';
+
+type Props = {
+  data?: Array<KeyValueListData>;
+  onClick?: () => void;
+  raw?: boolean;
+  longKeys?: boolean;
+  isContextData?: boolean;
+  isSorted?: boolean;
+};
+
+export type KeyValueListData = {
+  key: string;
+  subject: React.ReactNode;
+  value: string | null;
+  meta: Meta;
+};
+
+type Meta = {
```

Comment:
Will we need to have those types for every component? I wonder why we cannot introduce an additional `meta` argument to the existing keyValueList

Reply:
Afterwards I’ll check all the places where the Meta is being used and maybe I’ll move it to `src/sentry/static/sentry/app/types/*`  folder, so then we can reuse the Meta type...but for now I'd prefer to leave like that

Reply:
Having a reusable type definition would be nice as it prevents the various components from diverging.